ui/pages/confirmations/components/simulation-details/useSimulationMetrics.ts

```diff
@@ -67,6 +67,10 @@ export function useSimulationMetrics({
     setLoadingComplete();
   }
 
+  if(!simulationData) {
```

Comment:
We've other hooks below including a `useEffect` so would this condition best be added to the `shouldSkipMetrics` check inside the effect?

Reply:
Good call but `useIncompleteAssetEvent` is called regardless of that variable. 

Now I notice that `useIncompleteAssetEvent` also doesn't care if metrics enabled or not 😀

Reply:
I think the way we trigger `SimulationIncompleteAssetDisplayed` is getting send regardless of that variable, it's not tied to transaction. 

But now I noticed that `useIncompleteAssetEvent` not even cares if metrics are enabled or not 😄